test/populate_test/bdl_commands_test/heat_rejection_test.py

```diff
@@ -14,16 +22,108 @@ def setUp(self):
         self.rmd.doe2_version = "DOE-2.3"
         self.rmd.doe2_data_path = Config.DOE23_DATA_PATH
         self.heat_rejection = HeatRejection("Heat Rejection 1", self.rmd)
+        self.loop = CirculationLoop("Loop 1", self.rmd)
+        self.pump = Pump("Pump 1", self.rmd)
```

Comment:
I don't think pumps are needed to populate anything in heat_rejection, but instead we should add a test to the pump_test.py module to make sure that pump.loop_or_piping populates based on the HeatRejection.loop when a pump is assigned to the HeatRejection object